CONTRIBUTING.md

```diff
@@ -29,19 +29,32 @@ If you see an issue that you'd like to fix feel free to do so. If possible let u
 
 Skosmos PHP code should follow [PSR-12](https://www.php-fig.org/psr/psr-12/) style. To help achieve this, [PHP-CS-Fixer](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer), a tool to automatically fix and verify PHP code style issues, is included as a development dependency that is installed via Composer.  There is also a code style check in the GitHub Actions CI pipeline that verifies the code style compliance.
 
-You can set up a pre-commit hook to automate style checking with PHP-CS-Fixer for every git commit by adding the following script to the file `.git/hooks/pre-commit`, which should have execute permission set:
+We expect good coding practices also for front-end code. Among many others, we use [Standard JS](https://standardjs.com/) to validate the JavaScript code.
+
+You must set up a pre-commit hook to automate style checking with PHP-CS-Fixer for every git commit by adding the following script to the file `.git/hooks/pre-commit`, which should have execute permission set. Pre-commit hook for Standard JS validation should also be added (more details about Standard JS in the text below).
 
 ```bash
 #!/bin/bash
 
 set -e
 vendor/bin/php-cs-fixer fix --diff --dry-run src
+
+JS_FILES_TO_BE_CHECKED="/installation_folder/skosmos/*.js /installation_folder/skosmos/some_other_folder/*.js"
```

Comment:
Do we expect the developer to change this variable according to the location?

I think it would be better to just use a relative path. We only want to check the JS code of Skosmos itself, not other JS libraries for example under node_modules or Fuseki installed under tests/.

Since all Skosmos JS code has been removed on this branch, it's a little unclear what the paths should be, but we can assume that the location of JS code will remain the same as for Skosmos 2 - it will be placed under `resource/js/`.

I tested this command and it works (when I place a test file under resource/js/):

    npx standard resource/js/

So I suggest removing the JS_FILES_TO_BE_CHECKED declaration and just running the above command.

Reply:
I think it's possible that someone who contributes to the Skosmos front-end might want to place JS files somewhere other than the path we assume. It is also possible that there will be multiple folders to check later on. In my opinion, this is worth thinking about more carefully when we actually start to have JavaScript code to test. Paths can now be relative paths.